src/main/java/gg/projecteden/nexus/features/chat/commands/MessageCommand.java

```diff
@@ -26,6 +28,9 @@ void message(OfflinePlayer to, String message) {
 		if (isSelf(to))
 			error("You cannot message yourself");
 
+		if (MuteMenuUser.hasMuted(to, MuteMenuCommand.MuteMenuProvider.MuteMenuItem.MESSAGES))
```

Comment:
Can you [disable this setting](![image](https://user-images.githubusercontent.com/12417961/135560027-1d0cbf43-88a3-4393-8c2d-43eaab83dac1.png)) in Intellij and delete all `MuteMenuCommand.MuteMenuProvider.` so its just `MuteMenuItem.MESSAGES`